mage_ai/ai/hugging_face_client.py

```diff
@@ -0,0 +1,61 @@
+import json
+import os
+import re
+from typing import Dict
+
+import requests
+
+from mage_ai.ai.ai_client import AIClient
+from mage_ai.data_preparation.repo_manager import get_repo_config
+
+llama_model = "meta-llama/Llama-2-7b-hf"
+mistralai_model = "mistralai/Mistral-7B-Instruct-v0.1"
+
+headers = {
+    "Content-Type": "application/json"
+}
+
+"""
+This HuggingFaceClient is calling a Hugging Face Inference API endpoint
+with API endpoint and API token: https://huggingface.co/inference-endpoints.
+
+The model I tested and verified is mistral-7b-instruct-v0.1.
+"""
+
+
+class HuggingFaceClient(AIClient):
+    def __init__(self):
+        hf_config = get_repo_config().ai_config.get("hugging_face_config")
```

Comment:
nit: we prefer single quote in the code.

Reply:
Since you already define the AIConfig schema.
You can do `ai_config = AIConfig.load(config=get_repo_config().ai_config)`
Then you can use `ai_config.hugging_face_config` and `hr_config.huggingface_inference_api_token` to access the attributes.